smartsim/_core/utils/helpers.py

```diff
@@ -36,6 +36,35 @@
 
 from smartsim._core._install.builder import TRedisAIBackendStr as _TRedisAIBackendStr
 
+
+def unpack_db_identifier(db_id: str, token: str) -> t.Tuple[str, str]:
+    """Unpack the unformatted database identifier using the token,
+    and format for env variable suffix
+    :db_id: the unformatted database identifier eg. charizard_0
```

Comment:
I'll play the role of crotchety, no-fun @al-rigazzi here...but we should probably rename this :(